server/routerlicious/packages/local-server/.eslintrc.cjs

```diff
@@ -8,11 +8,24 @@ module.exports = {
 		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
 		"prettier",
 	],
+
 	parserOptions: {
 		"project": ["./tsconfig.json", "./src/test/tsconfig.json"],
 		"promise/catch-or-return": ["error", { allowFinally: true }],
 
 		// TODO: enable strict null checks in tsconfig and remove this override
 		"@typescript-eslint/prefer-nullish-coalescing": "off",
 	},
+	rules: {
+		// This package runs in node but also in the browser so we don't want it to import Node packages.
+		"import/no-nodejs-modules": ["error"],
+	},
```

Comment:
I think let's remove this bit too, since the base config already has this. We just need the override to allow test files to use nodejs modules.